plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Icon.js

```diff
@@ -100,8 +100,6 @@ Ext.define('NX.controller.Icon', {
     });
 
     // create the style sheet
-    // NOTE: This has issues on IE 11, forced compat via X-UA-Compatible meta tag
-    // NOTE: ... but may be better off rendering this server side?
     me.stylesheet = Ext.util.CSS.createStyleSheet(styles.join(' '), 'nx-icons');
```

Comment:
I wonder if the mentioned problems were related to how when IE sees the meta tag, it then has to start re-parsing the document again.

In case this re-parsing causes some sort of issue, there apparently is an option to return an HTTP header instead.

From [here](https://www.modern.ie/en-us/performance/how-to-use-x-ua-compatible):
> The best practice is an X-UA-Compatible HTTP Header. Adding the directive to the response header tells Internet Explorer what engine to use before parsing content begins.